bindings/wysiwyg-ffi/Cargo.toml

```diff
@@ -5,7 +5,7 @@ homepage = "https://gitlab.com/andybalaam/wysiwyg-rust"
 repository = "https://gitlab.com/andybalaam/wysiwyg-rust"
 description = "Swift and Kotlin bindings for wysiwyg-rust"
 keywords = ["matrix", "chat", "messaging", "composer", "wysiwyg"]
-license = "AGPL-3.0"
+license = "See license in README.md file in the root of this repository"
```

Comment:
Looks like `licence-file` should be used where there is a non-standard licence -> https://doc.rust-lang.org/cargo/reference/manifest.html#the-license-and-license-file-fields.

I think `licence` is meant to contain just SPDX format.

Reply:
Actually, that same website mentions this:

> The following files are always included:
> - The Cargo.toml file of the package itself is always included, it does not need to be listed in include.
> - A minimized Cargo.lock is automatically included if the package contains a binary or example target, see [cargo package](https://doc.rust-lang.org/cargo/commands/cargo-package.html) for more information.
> - If a [license-file](https://doc.rust-lang.org/cargo/reference/manifest.html#the-license-and-license-file-fields) is specified, it is always included.

So maybe we should create a `LICENSE-AGPLv3` file and change the contents in `LICENSE` so they mention both `LICENSE-APGLv3` and `LICENSE_COMMERCIAL`?

Reply:
Not sure I follow exactly, you mean create a `Licence.md` file and point to that, which details the licence information?

I think it's fine to point to `README.md` and then we are sticking to similar format across our repos(web and mobile repos both have this licence + copyright section now I think).